pkgs/tools/system/osquery/default.nix

```diff
@@ -0,0 +1,64 @@
+{ libcxxStdenv, lib, fetchurl, fetchFromGitHub, pkgconfig, git,
+  cmake, python3, which, flex, bison, perl, libunwind, openssl,
+  buildAws ? false, buildBpf ? true
+}:
+
+libcxxStdenv.mkDerivation rec {
+  pname = "osquery";
+  version = "4.9.0";
+
+  src = fetchFromGitHub {
+    owner = "facebook";
+    repo = pname;
+    rev = version;
+    fetchSubmodules = true;
+    sha256 = "sha256:0zr4fyw88jrg1z2cpy3gijmwjyr5qwdp5np1if6cy74rq6bdhfkw";
+  };
+
+  nativeBuildInputs = [python3 which git cmake pkgconfig flex bison perl];
```

Comment:
Maybe change `pkgconfig` to `pkg-config`? See https://github.com/NixOS/nixpkgs/pull/109887.